docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -2482,16 +2482,8 @@ alertmanager_client:
 # matchers for routes and inhibition rules, in silences, and in the labels for
 # alerts. It is recommended that all tenants run the `migrate-utf8` command in
 # mimirtool before enabling this mode. Otherwise, some tenant configurations
-# might fail to load. To identify tenants with incompatible configurations,
-# search Mimir server logs for lines containing `Alertmanager is moving to a new
-# parser for labels and matchers, and this input is incompatible`. To find
-# tenant configurations that are valid but contain ambiguous matchers, search
-# for log lines containing `Matchers input has disagreement`. Each log line
-# includes the invalid input, a suggestion on how to fix the input (excluding
-# ambiguous matchers, as these require manual correction), and the ID of the
-# affected tenant. You must run Mimir with debug-level logging enabled.
-# Otherwise, these lines aren't logged. For more information, refer to
-# https://prometheus.io/docs/alerting/latest/configuration/#label-matchers.
+# might fail to load. For more information, refer to
+# https://grafana.com/docs/mimir/<MIMIR_VERSION>/references/architecture/components/alertmanager/#enable-utf-8.
```

Comment:
I think we should include link text here instead of just the URL.

Reply:
Hi, do you mean like a Markdown URL? `[Enable UTF-8](https://grafana.com/docs/mimir/<MIMIR_VERSION>/references/architecture/components/alertmanager/#enable-utf-8)`?

Reply:
Exactly! You could even make it more specific like:
```
refer to [Enable UTF-8](https://grafana.com/docs/mimir/<MIMIR_VERSION>/references/architecture/components/alertmanager/#enable-utf-8) in the Grafana Mimir Alertmanager documentation.
```